tests/integration/test_script_runner.py

```diff
@@ -37,6 +37,22 @@ class BackendDependencies(enum.Enum):
         "util_script": "tests/integration/docusaurus/connecting_to_your_data/database/util.py",
         "extra_backend_dependencies": BackendDependencies.BIGQUERY,
     },
+    {
+        "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/pandas_azure_yaml_example/configured_asset_azure_data_connector.py",
+        "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+    },
+    {
+        "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/pandas_azure_python_example/configured_asset_azure_data_connector.py",
+        "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+    },
+    # {
+    #     "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/pandas_azure_yaml_example/inferred_asset_azure_data_connector.py",
+    #     "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+    # },
+    # {
+    #     "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/pandas_azure_python_example/inferred_asset_azure_data_connector.py",
+    #     "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+    # },
```

Comment:
As our Azure tests are less robust than what we have for S3 (due to the lack of a strong mocking library like `moto`), I believe it makes a lot of sense to have integration tests for both types of data connectors.

As such, I had organized our tests into directories. I've added the `InferredAssetAzureDataConnector` fixtures in our matrix as well as the relevant stubs to ensure adding tests is as simple as possible.

Reply:
Could talk a little bit more about your reasoning here? I agree that both type of data connectors would benefit from integration tests in the case of Azure (S3 would also benefit for that matter).  I'd like to know more about your reasoning behind adding the Azure tests in another folder. Is it because the `/cloud/` folder would become too cluttered otherwise?

Reply:
I'm not sold on the way I've implemented it here so please let me know if you have an alternative you think I should pursue!

I originally put it all in the `cloud/` dir but it got to be a bit cluttered like you noted. It is important to have an individual file per data connector type since we need a blank slate for each test (new GE install and all the other magic that happens in the `test_script_runner`). 

What we need to determine is the best file structure to organize our tests. What are your thoughts on this:
```
docusaurus
  connecting_to_your_data
    cloud
      s3
        pandas
          configured_python_example.py
          configured_yaml_example.py
          inferred_python_example.py
          inferred_yaml_example.py     
        spark
          ...
      azure
        ...
      gcs
        ...
```

Is there too much nesting (meaning we should get rid of the `pandas` subdir)? Or is there too listing nesting (meaning we should further split this out into additional subdirs `python` and `yaml`)?

I think we need some sort of change to have our test structure mirror the actual docs structure (like we do for GE tests). Maybe something we can connect on synchronously?